pkg/remote/aws/db_instance_supplier_test.go

```diff
@@ -72,10 +82,41 @@ func TestDBInstanceSupplier_Resources(t *testing.T) {
 					},
 				},
 			},
+			wantAlert: alerter.Alerts{},
+			err:       nil,
+		},
+		{
+			test:    "multiples mixed db",
+			dirName: "db_instance_multiple",
+			instancesPages: mocks.DescribeDBInstancesPagesOutput{
+				{
+					true,
+					&rds.DescribeDBInstancesOutput{
+						DBInstances: []*rds.DBInstance{
+							{
+								DBInstanceIdentifier: awssdk.String("terraform-20201015115018309600000001"),
+							},
+							{
+								DBInstanceIdentifier: awssdk.String("database-1"),
+							},
+						},
+					},
+				},
+			},
+			wantAlert: alerter.Alerts{},
+			err:       nil,
+		},
+		{
+			test:                "Cannot list db instances",
+			dirName:             "db_instance_empty",
+			instancesPagesError: awserr.NewRequestFailure(nil, 403, ""),
```

Comment:
Maybe it's a good option to start using mockery generated mock to return error instead of our own mocks for this case ?

Reply:
yeah it would have been easier for me but I didn'want to change all our test for this pr

Reply:
https://github.com/cloudskiff/driftctl/issues/161